experiments/ClimaEarth/components/land/climaland_bucket.jl

```diff
@@ -68,7 +68,7 @@ function BucketSimulation(
     if isnothing(shared_surface_space)
         domain = make_land_domain(depth; nelements, dz_tuple)
     else
-        domain = make_land_domain(shared_surface_space, depth)
+        domain = make_land_domain(shared_surface_space, depth; dz_tuple, nelements_vert = nelements[2])
```

Comment:
```suggestion
        domain = make_land_domain(shared_surface_space, depth; nelements_vert = nelements[2], dz_tuple)
```
(It doesn't really matter, just me being picky, you don't have to change it:)